protocol/messenger.go

```diff
@@ -6415,6 +6413,33 @@ func (m *Messenger) withChatClock(callback func(string, uint64) error) error {
 	return m.saveChat(chat)
 }
 
+func (m *Messenger) syncDeleteForMeMessage(ctx context.Context, rawMessageDispatcher RawMessageHandler) error {
+	deleteForMes, err := m.persistence.GetDeleteForMeMessages()
+	if err != nil {
+		return err
+	}
+
+	return m.withChatClock(func(chatID string, clock uint64) error {
```

Comment:
@qfrank do we really need `clock` since it is not used anywhere?

Reply:
fixed